src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -799,6 +799,11 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 DiagnosticLevel.Error,
                 "BCP139",
                 $"The root resource scope must match that of the Bicep file. To deploy a resource to a different root scope, use a module.");
+
+            public ErrorDiagnostic UnterminatedMultilineString() => new(
+                TextSpan,
+                "BCP140",
+                $"The multi-line string at this location is not terminated. Terminate it with \"'''\".");

```

Comment:
>The multi-line string at this location is not terminated. [](start = 18, length = 57)

I don't see a test that produces this error. Can you add a unit or integration test to cover it?

Reply:
Never mind - found it.

---
In reply to: [579478035](https://github.com/Azure/bicep/pull/1542#discussion_r579478035) [](ancestors = 579478035)

Reply:
Although could you add an integration test so it's more obvious where the error span lands?

---
In reply to: [579478358](https://github.com/Azure/bicep/pull/1542#discussion_r579478358) [](ancestors = 579478358,579478035)